package.json

```diff
@@ -1985,6 +1985,29 @@
 						"description": "Whether to show annotations against constructor, method invocations and lists that span multiple lines.",
 						"scope": "window"
 					},
+					"dart.closingLabelsPrefix": {
+						"type": [
+							"null",
+							"string"
+						],
```

Comment:
We should just make this `string` and use `// ` as the `default` value here instead. This makes it easier (or possible) to edit in the settings UI.